eth/chains/base.py

```diff
@@ -627,7 +647,7 @@ def estimate_gas(
         if at_header is None:
             at_header = self.get_canonical_head()
         with self.get_vm(at_header).state_in_temp_block() as state:
-            return self.gas_estimator(state, transaction)
+            return self.gas_estimator(state, transaction)               # type: ignore
```

Comment:
too much whitespace before comment. I also want to see if we can get rid of the `ignore` altogether but I haven't yet.